internal/common/transaction.go

```diff
@@ -25,5 +25,13 @@ type Transaction struct {
 	R                    *big.Int `json:"r"`
 	S                    *big.Int `json:"s"`
 	V                    *big.Int `json:"v"`
-	AccessListJson       string   `json:"access_list_json"`
+	AccessListJson       *string  `json:"access_list_json"`
```

Comment:
why it changed to be a pointer? or, actually, why it was not a pointer before? 😅

Reply:
to allow for null values. Otherwise it defaults to empty string